app/api/auth/routes.js

```diff
@@ -46,8 +46,8 @@ export default app => {
         body: {
           type: 'object',
           properties: {
-            username: { type: 'string' },
-            password: { type: 'string' },
+            username: { type: 'string', minLength: 1 },
+            password: { type: 'string', minLength: 1 },
```

Comment:
If I understand correctly, this would allow a password of one character, right?
Shouldn't a password be at least 8 characters in length (the current best practice)?

Reply:
Yes, but we don't have that policy right now, so if we put that validation in the login we will block many users from being able to log in, probably we will need to change the creation/edition of passwords, and then after some time maybe change it here.

Reply:
Yeah, I think we are mixing stuff :) We might definitively want to revisit our password policy. That's on user creation and password updates, where we decide what password a user can have.
This is the login. Technically, a user can try to login with any password, as long as they provide some.

Reply:
Ok, thanks for clarifying.